src/read.jl

```diff
@@ -41,6 +41,7 @@ function read_case(filepath)
     case["busid"] = convert(Array{Int,1}, mpc["bus"][:, 1])
     case["bus_demand"] = mpc["bus"][:, 3]
     case["bus_zone"] = convert(Array{Int,1}, mpc["bus"][:, 7])
+    case["bus_eiaid"] = convert(Array{Int,1}, mpc["bus"][:, 18])
```

Comment:
The idea is that this new column will be added to the base grid and/or add Scenario creation time, yeah?

Reply:
> The idea is that this new column will be added to the base grid and/or add Scenario creation time, yeah?

right

Reply:
We'll probably want to have some `try`/`catch` logic around this line so that if there's no 18th column in the bus table, `case["bus_eiaid"]` gets filled with zeros rather than throwing an error.